deployment/ansible/roles/ashlar.database/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: Create PostgreSQL super user
+  sudo_user: postgres
+  postgresql_user: name="{{ postgresql_username }}"
+                   password="{{ postgresql_password }}"
+                   role_attr_flags=SUPERUSER
+                   state=present
+
+- name: Create PostgreSQL database
+  sudo_user: postgres
+  postgresql_db: name="{{ postgresql_database }}"
+                 owner="{{ postgresql_username }}"
+
+- name: Add PostGIS extension
```

Comment:
I did see that awhile back, but probably not worth it just for this, since it works well. Most of our projects started enforcing 1.8, so I'd like to not force another bump so soon.